build.sbt

```diff
@@ -24,12 +21,24 @@ val sonatypeReleaseSettings = Seq(
   )
 )
 
+def artifactProducingSettings(supportScala3: Boolean) = Seq(
+  organization := "com.gu",
+  resolvers ++= Resolver.sonatypeOssRepos("releases"),
+  crossScalaVersions := Seq(scalaVersion.value) ++ (if (supportScala3) Seq("3.3.3") else Seq.empty),
```

Comment:
Note that Scala 2.12 is no longer here, and for artifacts that can support it (ie the `facia-json` artifacts) we now compile to Scala 3 as well.